src/test/resources/imp.kore

```diff
@@ -233,7 +233,7 @@ module MAP
 
   symbol merge{S1,S2}(Map{S1,S2}, Map{S1,S2}) : Map{S1,S2} []
 
-  symbol keys{S1,S2}(Map{S1,S2}) : Set{S1} []
+  symbol domains{S1,S2}(Map{S1,S2}) : Set{S1} []

```

Comment:
All files in `src/test/resources` are tested with the Kore parser;  therefore, if you changed this file, you also need to update the corresponding `.golden` file.  you can do that with `stack test --ta --accept`.

Reply:
Fixed.